docs_src/microservices/security/Ch-APIGateway.md

```diff
@@ -225,46 +225,88 @@ Comparing these two curl commands you may notice several differences.
 
 **Creating Access Token for API Gateway Authentication**
 
-If the EdgeX API gateway is not in use, a client can access and use any
-REST API provided by the EdgeX microservices by sending an HTTP request
-to the service endpoint. E.g., a client can consume the ping endpoint of
-the Core Data microservice with curl command like this: :
+The API gateway is configured to require authentiation prior to
+passing a request to a back-end microservice.
 
-    curl http://<core-data-microservice-ip>:48080/api/v1/ping
+It is necessary to create an API gateway user in order to
+satify the authentication requirement.  Gateway users
+are created using the proxy subcommand of the
+[secrets-config](secrets-config-proxy.1.md)
+utility.
+
+There are two ways to create a user, depending on how the API
+gateway is configured.
+
+**OAuth2 method**
+
+For OAuth2, a client ID and client secret are required:
+
+    docker-compose -p edgex -f docker-compose-ireland.yml run --rm --entrypoint /edgex/secrets-config edgex-proxy proxy adduser --token-type oauth2 --user _SOME_USERNAME_ --client_id _SOME_IDENTIFIER_ --client_secret _VERY_LONG_PASSWORD_
+
+User creation need only be done once.  Afterwards,
+an access token can be generated from the token endpoint on
+the API gateway.
 
```

Comment:
@beaufrusetta  and @bnevis-i  this needs to be addressed if we change the way due to the Admin API lock-down

Reply:
@beaufrusetta What's the new command I need here?